airsonic-main/src/main/java/org/airsonic/player/service/SettingsService.java

```diff
@@ -290,6 +292,10 @@ private static String getFileSystemAppName() {
     public static String getDefaultJDBCUrl() {
         return "jdbc:hsqldb:file:" + getAirsonicHome().getPath() + "/db/" + getFileSystemAppName();
     }
+    
+    public static int getDefaultUPnpPort() {
+        return Optional.ofNullable(System.getProperty("UPNP_PORT")).map(x -> Integer.parseInt(x)).orElse(DEFAULT_UPNP_PORT);
```

Comment:
For consistency with all other settings, could you use the same kind of casing (e.g. `UpnpPort`) as the other keys and store it at the top of the file in a constant?